app/ui/src/app/integration/list/status-detail.component.ts

```diff
@@ -85,6 +85,11 @@ export class IntegrationStatusDetailComponent implements OnChanges {
                                   current,
                                   total]);
       this.barWidth.width = ((current / total) * 100) + '%';
+      const consoleUrl = <string> this.configService.getSettings('consoleUrl');
+      // Check if the console URL is set properly, if not don't set up the log/event link
+      if (!consoleUrl || !consoleUrl.length) {
+        return;
+      }
       this.logUrl = undefined;

```

Comment:
Shouldn't this go up before the if ? Actually, it its unitialized it doesn't matter, but we should erase it if no consoleUrl is given.